packages/wrangler/src/core/wrap-command.ts

```diff
@@ -0,0 +1,117 @@
+import chalk from "chalk";
+import { readConfig } from "../config";
+import { FatalError, UserError } from "../errors";
+import { logger } from "../logger";
+import { printWranglerBanner } from "../update-check";
+import type { CommonYargsArgv } from "../yargs-types";
+import type {
+	BaseNamedArgDefinitions,
+	CommandDefinition,
+	HandlerArgs,
+	NamespaceDefinition,
+} from "./define-command";
+
+const betaCmdColor = "#BD5B08";
+
+export function wrapCommandDefinition(
+	def: CommandDefinition | NamespaceDefinition
```

Comment:
I'm a bit confused by the naming here. Why does a function called `wrapCommandDefinition()` take a `NamespaceDefinition`?